### PR TITLE
Feature/create bars page

### PR DESCRIPTION
**Pull request by sebastianwel**
Finished [US4](https://github.com/sebastianwel/MatchPal/issues/3). 
Needed to do some code-refactoring and data-uplifting to make the necessary mock-data available to all components. 
Besides that I created a BarsCard- and BarsList-Component as stated in the US. 

Please review and share feedback! :-)

### CODE REVIEW

**Comment by TmWgnr** [comment]
please pay attention to consistent naming.
otherwise your code looks good to me :)

File: pages/bars/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import AppHeader from "../../components/AppHeader/index.js";
```
Comment (actyvystom) [inline pages/bars/index.js:1]: Usually, you don't need to put the filename in the import statement if your component is defined in `index.js`, so e.g. 
```js
import AppHeader from "../components/AppHeader"
```
should suffice.

File: pages/index.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
 import MatchList from "../components/MatchList";
-import AppHeader from "../components/AppHeader/AppHeader";
-import AppFooter from "../components/AppFooter/AppFooter";
+import AppHeader from "../components/AppHeader/index.js";
```
Comment (actyvystom) [inline pages/index.js:2]: See above comment ;)

Comment (sebastianwel) [reply pages/index.js:2]: Oh understood, since it is an default export, right? 
I will adjust this in my next US :-)

Comment (sebastianwel) [reply pages/index.js]: Done :-)

**Review by FelineHuhn — APPROVED** [review]
LGTM now 🚀

**Pull request merged**